src/Illuminate/Pagination/AbstractPaginator.php

```diff
@@ -458,6 +458,21 @@ public function path()
         return $this->path;
     }
 
+    /**
+     * Resolve the query string or return the default value.
+     *
+     * @param  string|array|null  $default
+     * @return string
+     */
+    public static function resolveQueryString($default = null)
+    {
+        if (isset(static::$queryStringResolver)) {
+            return call_user_func(static::$queryStringResolver);
```

Comment:
```suggestion
            return (static::$queryStringResolver)();
```

Better performance.

Reply:
This follows the convention of other existing methods in the paginator such as `resolveCurrentPath`, `resolveCurrentPage`, etc.

Reply:
There are precedents of converting those calls, see https://github.com/laravel/framework/pull/29932. It would make Laravel better.

Reply:
Have included this change, thanks! 👍